nucypher/cli/deploy.py

```diff
@@ -140,7 +150,7 @@ def deploy(action,
 
     password = None
     if not hw_wallet and not blockchain.client.is_local:
-        password = get_nucypher_password(confirm=False)
+        password = get_client_password(confirm=False)
```

Comment:
@cygnusv `get_client_password` takes a `checksum` as a parameter and not `confirm`. Did you mean `get_nucypher_password`?
I'm getting the following traceback:
```
Traceback (most recent call last):
  File "/Users/derek/.local/share/virtualenvs/nucypher-M0kpWrFa/bin/nucypher-deploy", line 11, in <module>
    load_entry_point('nucypher', 'console_scripts', 'nucypher-deploy')()
  File "/Users/derek/.local/share/virtualenvs/nucypher-M0kpWrFa/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/Users/derek/.local/share/virtualenvs/nucypher-M0kpWrFa/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/Users/derek/.local/share/virtualenvs/nucypher-M0kpWrFa/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/derek/.local/share/virtualenvs/nucypher-M0kpWrFa/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/Users/derek/Documents/Github/repos/forks/david/nucypher/nucypher/cli/deploy.py", line 153, in deploy
    password = get_client_password(confirm=False)
  File "/Users/derek/Documents/Github/repos/forks/david/nucypher/nucypher/blockchain/eth/decorators.py", line 37, in wrapped
    params = inspect.getcallargs(func, *args, **kwargs)
  File "/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/inspect.py", line 1358, in getcallargs
    (f_name, kw))
TypeError: get_client_password() got an unexpected keyword argument 'confirm'
```

Reply:
Fixed, thanks!